scylla-cql/src/frame/response/event.rs

```diff
@@ -1,74 +1,117 @@
+//! CQL protocol-level representation of an `EVENT` response.
+
 use crate::frame::frame_errors::{
     ClusterChangeEventParseError, CqlEventParseError, SchemaChangeEventParseError,
 };
 use crate::frame::server_event_type::EventType;
 use crate::frame::types;
 use std::net::SocketAddr;
 
+/// Event that the server notified the client about.
 #[derive(Debug)]
 // Check triggers because all variants end with "Change".
 // TODO(2.0): Remove the "Change" postfix from variants.
 #[expect(clippy::enum_variant_names)]
 pub enum Event {
+    /// Topology changed.
     TopologyChange(TopologyChangeEvent),
+    /// Status of a node changed.
     StatusChange(StatusChangeEvent),
+    /// Schema changed.
     SchemaChange(SchemaChangeEvent),
```

Comment:
You could explain what is a topology change, and what is a status change (UP/DOWN vs ADD/REMOVE).

Reply:
If it is documented elsewhere then it is enough